SI/Sem.02/Pract.02/Task05.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
+int main() {
+	double sideA = 0, sideB = 0, sideC = 0;
+
+	std::cout << "Enter value for side a: ";
+	std::cin >> sideA;
+	std::cout << "Enter value for side b: ";
+	std::cin >> sideB;
+	std::cout << "Enter value for side c: ";
+	std::cin >> sideC;
+
+	if ((sideA + sideB <= sideC) || (sideA + sideC <= sideB) || (sideB + sideC <= sideA)) {
```

Comment:
Wrong conditions
https://www.wikihow.com/Determine-if-Three-Side-Lengths-Are-a-Triangle

Reply:
They're not wrong, because the cout in the if says "... is invalid...". I've just used de Morgan's Theorem and flipped the conditions. But I will say that it's definitely more confusing this way so I'll update it regardless.